__tests__/kata4.humanCatDogYears.test.js

```diff
@@ -1,6 +1,7 @@
 const { humanCatDogYears } = require('../src');
 
 describe('humanCatDogYears', () => {
+  // review
```

Comment:
You should be using 2 space indentation here, like you have done already inside the `it` callback.